Build failure in RyuJIT for ARM64

### Error Blob

```cpp
 /__w/1/s/src/coreclr/jit/emitarm64.cpp:23309:50: error: too few arguments to function call, expected 2, have 1
              emitDispElementIndex(emitGetInsSC(id));              // iii
              ~~~~~~~~~~~~~~~~~~~~                 ^
  /__w/1/s/src/coreclr/jit/emitarm64.cpp:20910:15: note: 'emitDispElementIndex' declared here
  void emitter::emitDispElementIndex(const ssize_t imm, const bool addComma)
```

```json
{
  "ErrorMessage": "emitter::emitDispElementIndex",
  "BuildRetry": false,
  "ErrorPattern": "",
  "ExcludeConsoleLog": true
}
```

cc: @TIHan 

### Reproduction Steps

_No response_
<!-- Known issue validation start -->
 ### Known issue validation
**Build: :mag_right:** https://dev.azure.com/dnceng-public/public/_build/results?buildId=554382
**Error message validated:** `emitter::emitDispElementIndex`
**Result validation:** :white_check_mark: Known issue matched with the provided build.
**Validation performed at:** 2/5/2024 11:50:31 PM UTC
<!-- Known issue validation end -->
<!--Known issue error report start -->
### Report
|Build|Definition|Step Name|Console log|Pull Request|
|---|---|---|---|---|
|[554389](https://dev.azure.com/dnceng-public/public/_build/results?buildId=554389)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/554389/logs/42)|dotnet/runtime#98006|
|[554383](https://dev.azure.com/dnceng-public/public/_build/results?buildId=554383)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/554383/logs/38)|dotnet/runtime#98008|
|[554380](https://dev.azure.com/dnceng-public/public/_build/results?buildId=554380)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/554380/logs/60)|dotnet/runtime#97487|
|[554370](https://dev.azure.com/dnceng-public/public/_build/results?buildId=554370)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/554370/logs/56)|dotnet/runtime#97234|
|[554382](https://dev.azure.com/dnceng-public/public/_build/results?buildId=554382)|dotnet/runtime|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/554382/logs/63)|dotnet/runtime#98008|
|[554369](https://dev.azure.com/dnceng-public/public/_build/results?buildId=554369)|dotnet/runtime|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/554369/logs/63)|dotnet/runtime#97234|
|[554363](https://dev.azure.com/dnceng-public/public/_build/results?buildId=554363)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/554363/logs/15)||
|[554352](https://dev.azure.com/dnceng-public/public/_build/results?buildId=554352)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/554352/logs/51)|dotnet/runtime#98007|
|[554348](https://dev.azure.com/dnceng-public/public/_build/results?buildId=554348)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/554348/logs/49)|dotnet/runtime#98007|
|[554351](https://dev.azure.com/dnceng-public/public/_build/results?buildId=554351)|dotnet/runtime|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/554351/logs/55)|dotnet/runtime#98007|
|[554347](https://dev.azure.com/dnceng-public/public/_build/results?buildId=554347)|dotnet/runtime|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/554347/logs/63)|dotnet/runtime#98007|
|[554279](https://dev.azure.com/dnceng-public/public/_build/results?buildId=554279)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/554279/logs/141)|dotnet/runtime#97934|
|[554329](https://dev.azure.com/dnceng-public/public/_build/results?buildId=554329)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/554329/logs/135)|dotnet/runtime#97972|
|[554324](https://dev.azure.com/dnceng-public/public/_build/results?buildId=554324)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/554324/logs/15)||
|[554302](https://dev.azure.com/dnceng-public/public/_build/results?buildId=554302)|dotnet/runtime|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/554302/logs/63)|dotnet/runtime#98006|
|[554299](https://dev.azure.com/dnceng-public/public/_build/results?buildId=554299)|dotnet/runtime|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/554299/logs/77)|dotnet/runtime#98006|
|[554278](https://dev.azure.com/dnceng-public/public/_build/results?buildId=554278)|dotnet/runtime|Build CoreCLR Runtime|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/554278/logs/590)|dotnet/runtime#97934|
#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|17|17|17|
<!--Known issue error report end -->

--- a/src/coreclr/jit/emitarm64.cpp
+++ b/src/coreclr/jit/emitarm64.cpp
@@ -23306,7 +23306,7 @@ void emitter::emitDispInsHelp(
             emitDispSveReg(id->idReg1(), id->idInsOpt(), true);  // ddddd
             emitDispSveReg(id->idReg2(), id->idInsOpt(), true);  // nnnnn
             emitDispSveReg(id->idReg3(), id->idInsOpt(), false); // mmm
-            emitDispElementIndex(emitGetInsSC(id));              // iii
+            emitDispElementIndex(emitGetInsSC(id), false);       // iii
             break;
 
         // <Pd>.B, <Pg>/Z, <Pn>.B, <Pm>.B

